MyFirstApplication/Services/Services.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
 using MyFirstApplication.Models;
-using MyFirstApplication.Repository;
 using Newtonsoft.Json;
```

Comment:
Seems like `Newtonsoft.Json` is still in use. Use the built in `System.Text.Json`. It has similar methods but a different class. Figure it out